chart/kubeapps/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: kubeapps
-version: 0.1.0
-appVersion: 1.0.0.alpha.4
+version: 0.1.1
+appVersion: 1.0.0.alpha.5
```

Comment:
The appVersion was mistakenly pointing to alpha4 while the one upstream uses alpha.5